scripts/build_vcpkg_deps.ps1

```diff
@@ -24,17 +24,17 @@
 
 # Parameters
 Param(
-    [Parameter(Mandatory=$true)][bool]$BuildAnimView,
-    [Parameter(Mandatory=$true)][string]$VcpkgTriplet,
-    [Parameter(Mandatory=$true)][string]$VcpkgCommitSha
+    [Parameter(Mandatory = $true)][bool]$BuildAnimView,
+    [Parameter(Mandatory = $true)][string]$VcpkgTriplet,
+    [Parameter(Mandatory = $true)][string]$VcpkgCommitSha
 )
 
 ################
 # Variables
 ################
 
 $anim_view_libs = "wxwidgets"
-$corsixth_libs = "ffmpeg", "freetype", "lua", "luafilesystem", "lpeg", "sdl2", "sdl2-mixer[libflac,mpg123,libmodplug,libvorbis]", "luasocket"
+$corsixth_libs = "ffmpeg", "freetype", "lua", "luafilesystem", "lpeg", "sdl2", "sdl2-mixer[libflac,mpg123,libmodplug,libvorbis]", "luasocket", "Catch2"
```

Comment:
port name is `catch2` all lowercase. This might matter if we use vcpkg in other environments in the future.